draft-ietf-tls-esni.md

```diff
@@ -509,8 +660,15 @@ if it were the "server_name" extension to finish the handshake. It
 SHOULD pad the Certificate message, via padding at the record layer,
 such that its length equals the size of the largest possible Certificate
 (message) covered by the same ESNI key. Moreover, the server MUST
-include the "encrypted_server_name" extension in EncryptedExtensions,
-and the value of this extension MUST match PaddedServerNameList.nonce.
+include the "encrypted_server_name" extension in EncryptedExtensions
+with the "response_type" field set to "esni_accept" and the "nonce"
+field set to PaddedServerNameList.nonce.
+
+If the server sends a NewSessionTicket message, the ticket MUST be
+ignored by servers not negotiating ESNI, including servers which do
+not implement this specification. This may be implemented by adding
+a new field to the server session state which earlier implementations
+cannot parse.
```

Comment:
I'm not sure how iluminating this hint about how to implement is. I would remove it.